USAGE.md

```diff
@@ -0,0 +1,386 @@
+
+If you're having trouble with your connection, e.g. because you switched networks or your device woke up from sleeping, you can easily reconnect to the last server with the `reconnect` or `r` option:
+
+`protonvpn reconnect`
+
+`protonvpn r`
+
+If you want to see the status and information of your current connection, you can use the `status` or `s` option, which doesn't require root:
+
+`protonvpn status`
+
+`protonvpn s`
+
+![status-example](https://i.imgur.com/5tm5nOJ.png)
+
+If you want to change different values that you've set during initialization, you can do this with the `configure` option, just follow the prompts to change your username/password, default protocol and so on:
+
+`protonvpn configure`
+
+![configuration-example](https://i.imgur.com/tHSkBxk.png)
+
+## Features
+
+### DNS Management
+
+#### DNS Leak Protection
+
+ProtonVPN-CLI features a DNS Leak Protection feature, which makes sure that you will use ProtonVPN's DNS Servers. This protects you from third parties (like your ISP) being able to see your DNS queries and therefore invading your privacy.
+
+ProtonVPN-CLI accomplishes this by updating the `/etc/resolv.conf` file when you connect to a VPN server and make sure that only ProtonVPN's DNS Server is written in this file. It will also backup the previous state of `/etc/resolv.conf` to revert all changes upon disconnection.
+
+Please note that when you change network (different Wifi) while being connected, `/etc/resolv.con` can be updated by your system. This can cause DNS leaks, so for best security, use `protonvpn reconnect` after changing network.
+
+**Enabling DNS Leak Protection**
+
+To enable DNS Leak Protection use the `protonvpn configure` command, then press `4` to choose DNS Management. Then press `1` to choose that you want to enable DNS Leak Protection.
+
+From now on your DNS queries are secured.
+
+#### Custom DNS
+
+If you always want to use a custom DNS Server when you connect to ProtonVPN, ProtonVPN-CLI allows you to do this as well. You can add up to 3 custom DNS Servers.
+
+**Enabling Custom DNS**
+
+To enable DNS Leak Protection use the `protonvpn configure` command, then press `4` to choose DNS Management. Then press `2` to choose that you want to configure custom DNS Servers. Now enter the IPs of up to 3 DNS Servers you want to use and confirm with Enter.
+
+#### Disabling DNS Management
+
+If you don't want ProtonVPN-CLI to do any changes to your DNS, you can do this as well. This will cause ProtonVPN-CLI to not touch `/etc/resolv.conf` and your device will always use the DNS servers configured by you or through your network.
+
+**Disabling any DNS management**
+
+To enable DNS Leak Protection use the `protonvpn configure` command, then press `4` to choose DNS Management. Then press `3` to disable any DNS management.
+
+### IPv6 Leak Protection
+
+ProtonVPN-CLI features an IPv6 Leak Protection feature. It makes sure that your IPv6 address doesn't get leaked when connecting to a ProtonVPN server.
```

Comment:
You used "feature" twice in a sentence :P It would sound nicer if you wrote "ProtonVPN-CLI features an IPv6 Leak Protector".